servlet-feature-pack/src/main/resources/modules/system/layers/base/javax/json/bind/api/main/module.xml

```diff
@@ -22,11 +22,7 @@
         <property name="jboss.api" value="public"/>
     </properties>
     <resources>
-        <artifact name="${javax.json.bind:javax.json.bind-api}">
-            <conditions>
-                <property-equal name="ee8.preview.mode" value="true"/>
-            </conditions>
-        </artifact>
+        <artifact name="${javax.json.bind:javax.json.bind-api}" />
```

Comment:
This is going to conflict with #11370. I'd rather see either #11370 merged first or this in a separate commit.

Reply:
I should have noted too that this change will require the change to use JSON-P 1.1 too as that is the dependency required for JSON-B 1.0. We could just add the two commits from #11370 to this PR too.

Reply:
I would be fine with merging #11370 first and then rebasing on top of that. What's preventing #11370 from being merged currently?